tests/integration/redis_cluster_tests.py

```diff
@@ -156,3 +170,91 @@ def test_pipeline(self):
         self.assertTrue(span_observer.on_start_called)
         self.assertTrue(span_observer.on_finish_called)
         self.assertIsNone(span_observer.on_finish_exc_info)
+
+    def test_metrics(self):
+        client_name = "redis_test"
+        for client_name_kwarg_name in [
+            "redis_client_name",
+            "client_name",
+        ]:
+            with self.subTest():
+                self.setup_baseplate_redis(
+                    redis_cluster_kwargs={
+                        client_name_kwarg_name: client_name,
+                    },
+                )
+                # Clear preometheus metrics
+                ACTIVE_REQUESTS.clear()
```

Comment:
This should probably be in setup so each test starts from a clean state

Reply:
I moved this to `tearDown` in #744. I will merge this before and then merge and remove these lines in that PR.